Automattic/wp-calypso
# Updates the way the Blaze Dashboard app loads the theme

I am changing the way the Blaze Dashboard app loads the theme. Now that the app is changing to support Woo Blaze, we need a way to define a theme dynamically depending on a configuration feature.

Now, the theme will be selected at runtime, at app load step.

## Proposed Changes

* Move CSS color variables definition to a JS file
* Load the correct theme at the app load step

## Testing Instructions
You will need a sandbox to test this easily. Ping me if you want to test this from a self-host site.

* Checkout this branch and in a console, navigate to `wp-calypso/apps/blaze-dashboard`
* Execute `yarn dev --sync`. This will sync the blaze app to your sandbox
* Move all the traffic from `widgets.wp.com` to your sandbox
* Open any self-hosted/pressable site, and navigate to Tools->Advertising
* Verify that the correct Jetpack Blaze theme is loaded (e.g., Black buttons, green selection background, etc).

## Pre-merge Checklist
